src/main/java/com/games/toki/pot/service/response/parser/CheapFlightsResponseParser.java

```diff
@@ -0,0 +1,32 @@
+package com.games.toki.pot.service.response.parser;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.games.toki.pot.service.response.CheapFlight;
+import reactor.core.publisher.Flux;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by mmundra on 1/16/19.
+ */
+public class CheapFlightsResponseParser {
```

Comment:
this serialiser should be included in webclient configuration
https://stackoverflow.com/questions/43769301/how-to-customize-springwebflux-webclient-json-deserialization